lib/private/User/User.php

```diff
@@ -536,13 +556,27 @@ public function setQuota($quota) {
 	}
 
 	public function getManagerUids(): array {
+		$user = $this->userSession->getUser();
```

Comment:
To make it cleared as there is already a lot of reference to user in this file.
```suggestion
		$connectedUser = $this->userSession->getUser();
```